rct229/rulesets/ashrae9012019/section19/section19rule20.py

```diff
@@ -0,0 +1,199 @@
+                    zone_supply_return_exhaust_relief_terminal_fan_power_dict_p[
+                        zone_id_b
+                    ]["zone_exhaust_fan_power"]
+                )
+                proposed_total_relief_fan_power += (
+                    zone_supply_return_exhaust_relief_terminal_fan_power_dict_p[
+                        zone_id_b
+                    ]["zone_relief_fan_power"]
+                )
+
+            total_modeled_fan_power_p = (
+                proposed_total_supply_fan_power
+                + proposed_total_return_fan_power
+                + proposed_total_exhaust_fan_power
+                + proposed_total_relief_fan_power
+            )
+            percent_of_total_supply_p = (
+                proposed_total_supply_fan_power / total_modeled_fan_power_p
+            )
+            percent_of_total_return_p = (
+                proposed_total_return_fan_power / total_modeled_fan_power_p
+            )
+            percent_of_total_exhaust_p = (
+                proposed_total_exhaust_fan_power / total_modeled_fan_power_p
+            )
+            percent_of_total_relief_p = (
+                proposed_total_relief_fan_power / total_modeled_fan_power_p
+            )
+
+            return {
+                "hvac_sys_total_supply_fan_power_b": hvac_sys_total_supply_fan_power_b,
+                "hvac_sys_total_return_fan_power_b": hvac_sys_total_return_fan_power_b,
+                "hvac_sys_total_exhaust_fan_power_b": hvac_sys_total_exhaust_fan_power_b,
+                "hvac_sys_total_relief_fan_power_b ": hvac_sys_total_relief_fan_power_b,
+                "expected_baseline_fan_power_supply": total_modeled_fan_power_b
+                * percent_of_total_supply_p,
+                "expected_baseline_fan_power_return": total_modeled_fan_power_b
+                * percent_of_total_return_p,
+                "expected_baseline_fan_power_exhaust": total_modeled_fan_power_b
+                * percent_of_total_exhaust_p,
+                "expected_baseline_fan_power_relief": total_modeled_fan_power_b
+                * percent_of_total_relief_p,
+            }
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            hvac_sys_total_supply_fan_power_b = calc_vals[
+                "hvac_sys_total_supply_fan_power_b"
+            ]
+            hvac_sys_total_return_fan_power_b = calc_vals[
+                "hvac_sys_total_return_fan_power_b"
+            ]
+            hvac_sys_total_relief_fan_power_b = calc_vals[
+                "hvac_sys_total_relief_fan_power_b"
+            ]
+            expected_baseline_fan_power_supply = calc_vals[
+                "expected_baseline_fan_power_supply"
+            ]
+            expected_baseline_fan_power_return = calc_vals[
+                "expected_baseline_fan_power_return"
+            ]
+            expected_baseline_fan_power_relief = calc_vals[
+                "expected_baseline_fan_power_relief"
+            ]
+            return (
```

Comment:
Missing exhaust fan power when taking out the values form calc_vals

Reply:
Good catch! Addressed.